vnext/Scripts/creaternwapp.cmd

```diff
@@ -113,13 +113,14 @@ for /f "delims=" %%a in ('npm show react@%R_VERSION% version') do @set R_VERSION
 
 set RNCLI_TEMPLATE=
 if not "x%RN_VERSION:nightly=%"=="x%RN_VERSION%" (
-    for /f "tokens=1 delims=-" %%a in ("%RN_VERSION%") do (
-        set "RNCLI_TEMPLATE=--template @react-native-community/template@^%%a"
-    )
+  @echo creaternwapp.cmd Override @react-native-community/template version
+  REM Do not change, this makes sure we always get a nightly template when still consuming a nightly RN and NOT a later "stable" template that may have been released
+  REM set RNCLI_TEMPLATE=--template "@react-native-community/template@^%RN_VERSION:~0,4%.0-"
+  set RNCLI_TEMPLATE=--template "@react-native-community/template@0.80.0-nightly-2025523-eb796ac"
```

Comment:
Since the old way (commented out above) no longer works with Node 22, we need to either:

1. Automatically update this `RNCLI_TEMPLATE` version to the closet available nightly template version by date (we'll need to write code to look it up) OR
2. Just add a comment here with a reminder that we need to manually update this with every integrate.